docs/framework/data/adonet/sql/sql-server-express-security.md

```diff
@@ -25,7 +25,7 @@ Microsoft SQL Server Express Edition (SQL Server Express) is based on Microsoft
   
 |||  
 |-|-|  
-|[Connecting to SQL Server Express](https://msdn.microsoft.com/library/ms165679.aspx) in SQL Server Books Online|Describes how to use SQL Server Express Edition on a network.|  
+|[Connecting to SQL Server Express](https://docs.microsoft.com/previous-versions/sql/sql-server-2005/ms165679(v=sql.90)) in SQL Server Books Online|Describes how to use SQL Server Express Edition on a network.|  
```

Comment:
This version is so old. SQL Server express version now is 2017. Better to remove it.

```suggestion
```